cypress/e2e/tests/administration/questionnaires/miscellaneous.test.ts

```diff
@@ -79,6 +90,34 @@ describe(["@tier3"], "Miscellaneous Questinnaire tests", () => {
         AssessmentQuestionnaire.backToQuestionnaire();
     });
 
+    it("Exports and imports existing questionnaire after updating name in file", function () {
+        // Automates polarion MTA-437
+        const targetNewName = "Cloud Native1";
+        const updatedFileName = "cloud-native-updated.yaml";
+        const fixturesPath = "cypress/fixtures/" + updatedFileName;
+
+        AssessmentQuestionnaire.import(cloudNativePath);
+        AssessmentQuestionnaire.export(cloudNative);
+        cy.readFile(cloudNativeDownloadPath).then((fileContent) => {
+            const updatedContent = AssessmentQuestionnaire.updateYamlContent(
+                fileContent,
+                cloudNative
+            );
+            cy.writeFile(fixturesPath, updatedContent);
+        });
+
+        cy.readFile(fixturesPath).then(() => {
+            AssessmentQuestionnaire.import(updatedFileName);
+        });
+        checkSuccessAlert(
+            alertTitle,
+            `Success alert:Questionnaire ${targetNewName} was successfully created.`,
+            true
+        );
+        AssessmentQuestionnaire.delete(cloudNative);
+        AssessmentQuestionnaire.delete(targetNewName);
```

Comment:
Removal should be moved to the `after()` section, clean up we are usually perform there until deletion is not the part of the test

Reply:
are specific for the test so maybe other tests want's to import cloudNative if i move the delete to after other tests can't import cloudNative